ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ModifyVulnReport/CreateVulnReportPage.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+    PageSection,
+    Title,
+    Divider,
+    Flex,
+    FlexItem,
+    Breadcrumb,
+    BreadcrumbItem,
+    Wizard,
+} from '@patternfly/react-core';
+
+import { vulnerabilityReportsPath } from 'routePaths';
+import useReportFormValues from 'Containers/Vulnerabilities/VulnerablityReporting/forms/useReportFormValues';
+
+import PageTitle from 'Components/PageTitle';
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import ReportParametersForm from 'Containers/Vulnerabilities/VulnerablityReporting/forms/ReportParametersForm';
+import DeliveryDestinationsForm from 'Containers/Vulnerabilities/VulnerablityReporting/forms/DeliveryDestinationsForm';
+import ReportReviewForm from 'Containers/Vulnerabilities/VulnerablityReporting/forms/ReportReviewForm';
+import useCreateReport from '../api/useCreateReport';
+import ReportFormWizardFooter from './ReportFormWizardFooter';
+import ReportFormErrorAlert from './ReportFormErrorAlert';
+
+const wizardStepNames = [
+    'Configure report parameters',
+    'Configure delivery destinations (Optional)',
+    'Review and create',
+];
+
+function CreateVulnReportPage() {
+    const history = useHistory();
+
+    const { formValues, setFormFieldValue, clearFormValues } = useReportFormValues();
+    const { data, isLoading, error, createReport } = useCreateReport();
+
+    // When report is created, navigate to the vuln reports page
+    // @TODO: We want to change this in the future to navigate to the read-only report page to verify the details
+    useEffect(() => {
+        if (data) {
+            clearFormValues();
+            history.push(vulnerabilityReportsPath);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [data]);
+
+    function onCreate() {
+        createReport(formValues);
+    }
+
+    const wizardSteps = [
+        {
+            name: wizardStepNames[0],
+            component: (
+                <ReportParametersForm
+                    title={wizardStepNames[0]}
+                    formValues={formValues}
+                    setFormFieldValue={setFormFieldValue}
+                />
+            ),
+        },
+        {
+            name: wizardStepNames[1],
+            component: (
+                <DeliveryDestinationsForm
+                    title={wizardStepNames[1]}
+                    formValues={formValues}
+                    setFormFieldValue={setFormFieldValue}
+                />
+            ),
+        },
+        {
+            name: wizardStepNames[2],
+            component: <ReportReviewForm title={wizardStepNames[2]} formValues={formValues} />,
```

Comment:
I added a new prop called `title` for each of these wizard components because the last step name, for example, will be different depending on the flow (ie. create, edit, clone)